### DISCUSSION TITLE
Fprime File operations - Demo

### DISCUSSION THREAD

**Discussion by satzguyz** [nasa/fprime#1377] [Q&A] [answered]
Hello,

I am running the GDS after the default compilation and build of Ref applications. I tried to create a folder, using the Web UI.
![image](https://user-images.githubusercontent.com/103112583/162777167-15c72c27-7010-4c37-b863-e1a679440244.png)

Where can I see the creation of new folder ?

**Comment by LeStarch** [comment, marked as answer]
That command would run on the "spacecraft".  If you ran on your local machine, it is the "spacecraft" in terms of your demo.

Since a full path was not specified, the folder would be created relative to the working directory where you ran F´.   If you ran `fprime-gds` to launch F´, then the folder would have been created as a subfolder to the one in ehich you ran `fprime-gds`.
